### ISSUE TITLE
TO DO in Beta#3 - Last Release

### ISSUE THREAD

**Issue by theraw** [theraw/xtreamui-on-docker#9]
1. Custom PHP Build *(current is default apt-get install php it works but we need custom build)*
2. Nginx Admin panel Conf CleanUP + Security Checkings
3. Nginx RTMP optimize/cleanup/security checkings.
5. Container Cleanup reducing size.
6. Volume Mount, in order to share videos files host<->container we can easily mount a folder.
7. **Maybe** Proxy Setup in order to use `example.com` with free ssl certs.
8. Cronjob automatically setup no need for manual setup.

Ask of anything what feature you'll like to see in next release because `Beta#3` is last release from me since XtreamUI dev (GTA) stopped supporting this project.

**Comment by GaiusBaltar790** [comment]
Good stuff really, I was doing same but you faster. 
May be you could share also Dockerfile?
Thanks

**Comment by theraw** [comment]
@GaiusBaltar790 no. not because i don't want but because i don't have one, this took some of my hours to build and is continuous work not something that you can build at once on a dockerfile.

the way how i work with docker containers is having a base image for each distro then i use it for my new builds that means my dockerfiles are as simple as https://docs.docker.com/engine/examples/running_ssh_service/

Publishing this in a dockerfile will take ppl hours to compile 2x nginx +php and other installations also can cause problems as builds can fail or services can fail to start etc and i'm not willing to deal with those things sharing a ready to go container seems easier in my part that's the reason i don't like dockerfiles.

**Comment by GaiusBaltar790** [comment]
No problem at all. thanks for the image, I will try it soon.
Cheers

**Comment by dberuben** [comment]
Where i can found the database root password (mysql)?

**Comment by theraw** [comment]
into home of the root user of your container.

**Comment by dwopetweens** [comment]
i have a problem when i add a stream its stopped suddenly .what do i need to change ?

**Comment by theraw** [comment]
there could be millions of reasons why your stream stops. 

And i'm not xtream-ui support...

**Comment by dwopetweens** [comment]
Does it work for you on streams
